giraffe/src/components/TableCell.tsx

```diff
@@ -164,19 +169,23 @@ const getContents = (props: Props): string => {
   } = props
   const {decimalPlaces} = properties
 
+  if (!data || (isString(data) && isBlank(data))) {
+    return ''
```

Comment:
`return String(data)`

Let's just give back what the value was that caused this edge case.